add-api/client/src/components/Header.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Header: React.FC = () => {
+  return (
+    <div className="text-center mb-4">
+      <h1>🦍 Gorilla: API Zoo Data Converter</h1>
+      <div className="text-center">
+        <p className="lead">Easily give your API information to Gorilla API Store</p>
```

Comment:
Maybe reword this to "Easily **add** your API information to Gorilla API Store"?

@ShishirPatil Also, should we use "Gorilla API Store" or "Gorilla API Zoo"? I think it would be good to have a consistent name.